packages/db/src/search/field-mapping.ts

```diff
@@ -123,6 +123,18 @@ const collectionMapping : { [key: string]: Function } = {
   updatedAt: (value?: string) => ({
     updated_at: value && new Date(Number(value)),
   }),
+  reportToEms: (value?: string) => ({
```

Comment:
Sorry, keep forgetting to add this comment. The feature branch has a test file `packages/db/tests/search/test-field-mapping.js`. As you are adding some field mappings, do you not need to add any tests to that file for these?

Reply:
Ok, I missed the actual test before, but it is in that file I referenced on line 104. I think that test should include some of these fields you added, like report_to_ems, sample_file_name, and url_path